difflint/lib.py

```diff
@@ -1,27 +1,15 @@
+import argparse
 from datetime import datetime
 from difflib import unified_diff
 from io import StringIO
-from os import getcwd, listdir, unlink
-from os.path import exists, isfile, join, splitext
-from pkg_resources import resource_filename
+import os
 import re
-from subprocess import call, check_call, check_output, CalledProcessError, DEVNULL
+from subprocess import call, check_output
 from sys import stderr
 
-LOG_FILE = "lintdiff.log"
-LINT_PATH = resource_filename(__name__, 'data/lint.sh')
-
-
-class LintOutput(object):
-    def __init__(self, output, warning_boolean):
-        self.output = output
-        self._warnings_present = warning_boolean  # private
+from .lint import get_missing_linters, lint
 
-    def has_warnings(self):
-        return self._warnings_present
-
-    def get_split_output(self):
-        return self.output.splitlines(keepends=True)
+LOG_FILE = "lintdiff.log"
```

Comment:
nit: Recommend consistent use of single quotes over mixing single and double quotes (multiple instances, pre-existing defect).